src/i18n.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { i18n } from '@lingui/core'
+import { I18nProvider } from '@lingui/react'
+import { detect, fromUrl } from '@lingui/detect-locale'
+import { ReactNode, useEffect } from 'react'
+
+export const locales = ['en', 'it']
+export const defaultLocale = 'en'
+
+locales.map((locale) => i18n.loadLocaleData(locale, { plurals: () => null }))
```

Comment:
please explain this line in a comment, and since you aren't using the return value it should be `forEach`. e.g. why can't this be done with the locale is detected?

also i guess we don't care about using plurals?

Reply:
I think we'll want to add plurals soon enough. I didn't needit yet. It increases the bundle size.